packages/aws-cdk/README.md

```diff
@@ -362,3 +362,11 @@ The following environment variables affect aws-cdk:
 
 - `CDK_DISABLE_VERSION_CHECK`: If set, disable automatic check for newer versions.
 - `CDK_NEW_BOOTSTRAP`: use the modern bootstrapping stack.
+
+
+#### Supported credential providers
```

Comment:
This is not complete I think. Any special reason this block was added?

Reply:
I suspect this was added because of the PR validation.

I've added an exemption label to the PR so it shouldn't require it anymore. Remove this? I'll take a task to properly document the entire credentials handling for later.